src/main/java/de/unijena/cheminf/mortar/gui/util/GuiUtil.java

```diff
@@ -252,7 +251,7 @@ public static void guiBindControlSizeToParentPane(Pane aParentPane, Control aChi
      * @return GUI input pattern for integer values
      */
     public static Pattern getIntegerPattern(){
-        return Pattern.compile("-?(([1-9][0-9]*)|0)?");
+        return Pattern.compile("-?(([1-9]\\d*)|0)?");
```

Comment:
Be careful with this, [0-9] is not always equivalent to \d

Reply:
Thank you for noticing! I think I'm going to reverse this anyway since [0-9] is more straightforward for people not dealing with regex every day (i.e. me and potential students)...